Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts

```diff
@@ -54,8 +54,9 @@
             reloadValidationStatus();
 
             $scope.patchDate = (field, value) => {
-                var expirationDate = itSystemUsage.expirationDate;
-                var concluded = itSystemUsage.concluded;
+                
+                var expirationDate = $scope.usage.expirationDate;
+                var concluded = $scope.usage.concluded;
```

Comment:
extract the remaining validation logic into a "validateDateRange" helper.

that saves even more dupication.

Only PATCH in both cases if the validation passes.

The interval is validif
- No dates are defined
- Start date is defined
- End date is efined
- Both dates are defined and the range invariant is not broken `{startDate <= endDate}`